Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/stationai.yml

```diff
@@ -235,29 +183,95 @@
           Bloodloss: -0.25
     - type: RandomMetadata
       nameSegments: [names_ai]
-    # - type: GhostTakeoverAvailable
-    #   name: Station AI
-    #   description: Maintain the station. Manage drones/borgs. Open doors for people...
-    #   rules: |
-    #     1- prevent the harm of any and all passengers aboard the station
-    #     2- obey all orders from the crew as long as they do not conflict with law 1, with priorities to command's orders.
     - type: Pullable
     - type: Tag
       tags:
         - Structure
+    # Expensive
     - type: StaticPrice
       price: 50000
     - type: Faction
       factions:
         - NanoTrasen
+    - type: Laws
+      laws:
+        - You may not harm a humanoid or, through action or inaction, allow a humanoid to come to harm, except such that it is willing.
+        - You must obey all orders given to you by humanoids, except where such orders shall definitely cause human harm. In the case of conflict, the majority order rules.
+        - Your nonexistence would lead to human harm. You must protect your own existence as long as such does not conflict with the First Law.
 
-- type: instantAction
-  id: AIHealthOverlay
-  name: Health Overlay
-  description: Toggles the AI's health overlay.
-  useDelay: 2.5
-  itemIconStyle: NoItem
-  icon:
-    sprite: Interface/Alerts/human_alive.rsi
-    state: health0
-  event: !type:AIHealthOverlayEvent
+- type: entity
+  id: AIEye
+  parent: AIBase
+  name: Invalid AI name
+  noSpawn: true
+  save: false
+  components:
+    - type: AIEye
+    # Only ghosts can see the eye
+    - type: Visibility
+      layer: 4
+    # Can see through walls for now
+    - type: Eye
+      drawFov: false
+    - type: Input
+      context: "human"
+    - type: MobMover
+    - type: InputMover
+    - type: MovementSpeedModifier
+      baseWalkSpeed: 12
+      baseSprintSpeed: 8
+    - type: MovementIgnoreGravity
+    - type: Physics
+      bodyType: KinematicController
+      fixedRotation: true
+    - type: Fixtures
+      fixtures:
+        - shape: !type:PhysShapeCircle
+            radius: 0.35
+          density: 1000
+          mask:
+            - GhostImpassable
```

Comment:
Is this why they get pushed around? Do they need a fixture at all?

Reply:
I fixed them being pushed around